android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -84,6 +93,22 @@ public String getName() {
         return map;
     }
 
+    @RequiresApi(Build.VERSION_CODES.O)
+    private void createChannel(ReactApplicationContext context) {
+        NotificationManager
+            mNotificationManager =
+            (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
```

Comment:
There's no need to break this into 3 lines

Reply:
Right